identity/multiaddress.go

```diff
@@ -154,6 +154,11 @@ func (multiAddress *MultiAddress) UnmarshalStringJSON(data []byte) error {
 	return nil
 }
 
+// IsEmpty checks if the multiAddress is nil.
+func (multiAddress *MultiAddress) IsEmpty() bool {
+	return (multiAddress == nil || multiAddress.baseMultiAddress == nil)
```

Comment:
What about the non-base address? Strings can be checked against the empty string `""`.